<a href="/vibhutisawant/taro-ui/next/patch/Makefile_containerd.diff">Moved Permanently</a>.

diff --git a/Makefile b/Makefile
index ac4277c..b70d290 100644
--- a/Makefile
+++ b/Makefile
@@ -14,7 +14,7 @@
 
 include common/common.mk
 
-BUILD_IMAGE=ubuntu:focal
+BUILD_IMAGE=ecos0003:5000/jenkins_slave_rhel:8.4
 BUILD_TYPE=$(shell ./scripts/deb-or-rpm $(BUILD_IMAGE))
 BUILD_BASE=$(shell ./scripts/determine-base $(BUILD_IMAGE))
 
@@ -93,7 +93,6 @@ build:
 	@if [ -z "$(BUILD_TYPE)" ]; then echo "Invalid build image $(BUILD_IMAGE) no build type found"; exit 1; fi
 
 	@set -x; DOCKER_BUILDKIT=1 docker build \
-		--pull \
 		--build-arg GOLANG_IMAGE="$(GOLANG_IMAGE)" \
 		--build-arg BUILD_IMAGE="$(BUILD_IMAGE)" \
 		--build-arg BASE="$(BUILD_BASE)" \
